src/rust/ensogl/src/display/shape/text/text_field/cursor.rs

```diff
@@ -624,16 +620,39 @@ mod test {
         let initial_cursors   = vec![initial_cursor];
         let new_position      = TextLocation {line:1,column:10};
 
-        let mut fonts      = FontRegistry::new();
-        let properties     = TextFieldProperties::default(&mut fonts);
-        let mut content    = TextFieldContent::new(text,&properties);
-        let mut navigation = CursorNavigation::new(&mut content);
+        let mut fonts       = FontRegistry::new();
+        let properties      = TextFieldProperties::default(&mut fonts);
+        let content         = &mut TextFieldContent::new(text,&properties);
+        let selecting       = false;
+        let text_field_size = properties.size;
+        let mut navigation  = CursorNavigation{content,text_field_size,selecting};
         let mut cursors = Cursors::mock(initial_cursors.clone());
         cursors.navigate_all_cursors(&mut navigation,LineEnd);
         assert_eq!(new_position, cursors.first_cursor().position);
         assert_eq!(new_position, cursors.first_cursor().selected_to);
     }
 
+    #[wasm_bindgen_test(async)]
+    async fn page_scrolling() {
+        ensogl_core_msdf_sys::initialized().await;
+        let text              = "1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n12\n13\n14\n15\n16\n17\n18\n19";
+        let initial_cursor    = Cursor::new(TextLocation::at_document_begin());
+        let initial_cursors   = vec![initial_cursor];
+        let expected_position = TextLocation {line:6,column:0};
+
+        let mut fonts  = FontRegistry::new();
+        let properties = TextFieldProperties::default(&mut fonts);
+        assert_eq!(properties.size, Vector2::new(100.0,100.0));
```

Comment:
```suggestion
        let mut properties = TextFieldProperties::default(&mut fonts);
        properties.size = Vector2::new(100.0,100.0);
```
I think we should just set required size instead of counting for defaults.